CHANGELOG.md

```diff
@@ -15,6 +15,99 @@ and includes an additional section for migration notes.
 
 
 ## [Unreleased]
+
+
+### Added
+- *ORCA-161* Added dead letter queue and cloudwatch alarm terraform code to recovery SQS queue.
+
+### Changed
+- *ORCA-161* Changed staged recovery SQS queue type from FIFO to standard queue.
+
+### Migration Notes
+
+- The user should update their `orca.tf`, `variables.tf` and `terraform.tfvars` files with new variable. The following required variable has been added:
+  - db_admin_username (defaults to "postgres")
```

Comment:
Shouldn't this be `dead_letter_queue_topic_subscription_email`? As a side note, @AndrewDorn  and @rizbi-hassan how do you feel about shortening the name of the variable to something like `dlq_subscription_email`?

Reply:
yes my bad.. removed and changed variable name to dlq_subscription_email